src/pages/account/organization/new/content.component.js

```diff
@@ -0,0 +1,60 @@
+import React, { useContext, useEffect, useState } from 'react';
+import { useRouter } from 'next/router';
+import { Skeleton } from 'antd';
+
+import Audit from './audit';
+import Banner from './banner';
+import Form from './form';
+import { AUDIT_STATUS } from './audit/audit.constants';
+import { SplitLayout } from 'layouts';
+import { MeContext, NavContext } from 'context';
+import Blank from 'components/Blank';
+
+const CreateOrganization = () => {
+  const router = useRouter();
+  const { meData, mutateMe, isMeValidating } = useContext(MeContext);
+  const { login } = useContext(NavContext);
+  const [showForm, setShowForm] = useState(false);
+
+  useEffect(() => {
+    setShowForm(!(meData?.org || meData?.org_enroll));
+  }, [meData]);
+
+  const status = meData?.org ? AUDIT_STATUS.PASS : meData?.org_enroll?.audit_status;
+  const rejectReason = meData?.org_enroll?.audit_reason;
+
+  const resetForm = () => setShowForm(true);
+  const pushOrgHome = () => router.push(`/orgs/${meData?.org?.slug}/members`);
+
+  if (!meData) {
+    if (isMeValidating) {
+      return (
+        <Blank>
+          <Skeleton active />
+        </Blank>
+      );
+    } else {
+      login();
+      return <></>;
+    }
+  }
```

Comment:
The pattern has been repeated on both the Invitation page and here. Could refactor it later.